src/flynt/cli.py

```diff
@@ -191,7 +194,18 @@ def run_flynt_cli(arglist: Optional[List[str]] = None) -> int:
         )
         print(converted)
         return 0
-
+    if "-" in args.src:
+        if len(args.src) > 1:
+            parser.error("Cannot use '-' with other arguments")
```

Comment:
Did you mean to say that it can't use `-` with other file paths? I guess arguments like line length, aggressive, etc. should still work?

Reply:
I've applied a fix to wording and merged, let me know if you think I've misunderstood something here.

Reply:
Yeah, I guess that would've been a better way to phrase that! 😁